larsim/EventGenerator/PhotonGen_module.cc

```diff
@@ -152,9 +169,23 @@ namespace evgen {
     fYmax = CryoBounds.MaxY();
     fZmin = CryoBounds.MinZ();
     fZmax = CryoBounds.MaxZ();
-    std::cout << "Cryo Boundaries:" << std::endl;
+    //Initial default boundaries---
+    std::cout << "\n\nINITIAL Cryo Boundaries:" << std::endl;
+
     std::cout << "Xmin: " << fXmin << " Xmax: " << fXmax << " Ymin: " << fYmin << " Ymax: " << fYmax
               << " Zmin: " << fZmin << " Zmax: " << fZmax << std::endl;
+
+    fXmin = fBminX;
+    fXmax = fBmaxX;
+    fYmin = fBminY;
+    fYmax = fBmaxY;
+    fZmin = fBminZ;
+    fZmax = fBmaxZ;
```

Comment:
See above comment.  These assignments should only be made when the configuration parameter is provided.  Otherwise, the values of `fXmin`, `fXmax`, etc. should be set to `CryoBounds.MinX()`, etc.